OpenSim/Common/DataTable.h

```diff
@@ -0,0 +1,268 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  DataTable.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
```

Comment:
Thanks. 
I don't like stamping my name on all the files I touch. I am not looking for anonymity. I know git keeps track of authors and that is good enough for me. I am completely fine if future contributors to these files put their name on it. I am just not going to do it.
If someone has questions regarding these files, he/she should post it to the whole group.